usagov/vote-gov
# Vote-112 -> French Accessibility Page

Ticket: [Vote-112](https://cm-jira.usa.gov/browse/VOTE-112)
Purpose:  This PR will add the french translation for the accessibility page
Testing:

1. pull down branch
2. run npm run start
3. visit french translation and check that the new accessibility page loads and there are no major regressions
